### DISCUSSION TITLE
Obtaining ground truth bbox coordinates

### DISCUSSION THREAD

**Discussion by AlexanderGeorge102934** [ultralytics/ultralytics#13418] [Q&A]
I am training my yolov8 model for SOD and I wanted to know how do I obtain the ground truth bounding box coordinates so that I can find the Euclidian distance between my predicted bounding box and my ground truth box

**Comment by glenn-jocher** [comment]
To obtain ground truth bounding box coordinates for your YOLOv8 model training, you'll need to prepare your dataset with annotations that include these coordinates. Typically, these annotations are stored in a format like JSON, XML, or plain text, where each object in an image is labeled with a bounding box specified by coordinates (e.g., x_min, y_min, x_max, y_max).

Once you have your dataset annotated, during training or inference, you can access these ground truth coordinates from your dataset loader or the annotation files directly. For calculating the Euclidean distance between predicted and ground truth bounding boxes, you can use a simple mathematical formula to measure the distance between the centers or any two corresponding points of the bounding boxes.

If you need further guidance on how to structure your annotations or implement the distance calculation, please let me know!
